conans/cli/commands/__init__.py

```diff
@@ -32,3 +33,35 @@ def default(self, o):
 def json_formatter(data):
     myjson = json.dumps(data, indent=4, cls=ConanJSONEncoder)
     return myjson
+
+
+def add_log_level_args(subparser):
+    subparser.add_argument("-v", "--output-level",  default="status", nargs='?',
+                           help="Level of detail of the output")
+    subparser.add_argument("--logger", action="store_true",
+                           help="Show the output with log format, with time, type and message.")
+
+
+def process_log_level_args(args):
+    from conans.cli import output
+    from conans.cli.output import LEVEL_QUIET, LEVEL_ERROR, LEVEL_WARNING, LEVEL_NOTICE, \
+        LEVEL_STATUS, LEVEL_VERBOSE, LEVEL_DEBUG, LEVEL_TRACE
+
+    levels = {None: LEVEL_VERBOSE,  # -v
+              "verbose": LEVEL_VERBOSE,  # -vverbose
+              "v": LEVEL_DEBUG,  # -vv
+              "debug": LEVEL_DEBUG,  # -vdebug
+              "vv": LEVEL_TRACE,  # -vvv
+              "trace": LEVEL_TRACE,  # -vtrace
+              "status": LEVEL_STATUS,  # -vstatus
+              "notice": LEVEL_NOTICE,  # -vnotice
+              "warning": LEVEL_WARNING,  # -vwaring
+              "error": LEVEL_ERROR,  # -verror
+              "quiet": LEVEL_QUIET  # -vquiet
+              }
```

Comment:
just a minor detail, but I think if would be useful when checking the code seeing these arguments ordered by verbosity and with the correspondence with the numeric level

Reply:
done